R/get_delay_estimate.R

```diff
@@ -94,5 +94,17 @@ get_delay_estimate <- function(triangle,
   # delay distribution
   pmf <- colSums(expectation) / sum(expectation)
 
+  if (pmf[1] == 0) {
+    cli_warn(
+      message =
+        c(
+          "Value of delay PMF at delay = 0 is 0. This delay PMF will not be ",
```

Comment:
I think here we may want to suggest some action a user should/can take? 

From a separation of concerns perspective do we need this warning here as having a zero is a totally valid delay and it only becomes a problem when it is used for nowcasting?

Reply:
Sure I guess its a bit overkill, can remove.